src/pages/Main/Main.js

```diff
@@ -1,7 +1,189 @@
 import React from 'react';
+import Slider from 'react-slick/lib/slider';
+import styled from 'styled-components';
+import { MAINCAROUSEL, PROJECTCAROUSEL } from './carouselData';
+import Card from '../../components/Card/Card';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import banner from '../../assets/wecodebanner.png';
 
 const Main = () => {
-  return <div>메인페이지</div>;
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    autoplay: true,
+    speed: 1000,
+    nextArrow: <NextArrow top="top" color="color" />,
+    prevArrow: <PrevArrow top="top" color="color" />,
```

Comment:
boolean 속성 하나만 있어도 될 것 같습니다.
```suggestion
    nextArrow: <NextArrow isTop />,
    prevArrow: <PrevArrow isTop />,
```